lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +19,94 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas: 
```

Comment:
You've probably already figured this out but named grid areas aren't supported in IE11

Reply:
> You've probably already figured this out but named grid areas aren't supported in IE11

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/344242f37afb43e9271a20254a4020d4726dfb66